src/shared/themes/index.js

```diff
@@ -5,11 +5,37 @@ export const dstTheme = createTheme({
     primary: {
       main: "#4F5F30",
     },
-    text: {
-      primary: "#FFFFFF",
+  },
```

Comment:
why not make this one theme and wrap the whole app with it?

Reply:
makes sense, I've removed the dstTheme 2 & I'm currently removing the unnecessary ThemeProviders